metrics/register.go

```diff
@@ -0,0 +1,14 @@
+package metrics
```

Comment:
I would prefer to keep all the init code inside of this file so that trash does not mark the other directories as unused and remove them.

Reply:
Sure let me update